src/internal/FloatingMenu.js

```diff
@@ -318,21 +318,27 @@ class FloatingMenu extends React.Component {
   _getChildrenWithProps = () => {
     const { styles, children } = this.props;
     const { floatingPosition: pos } = this.state;
-    return !pos
-      ? children
-      : React.cloneElement(children, {
-          style: Object.assign(
-            {
-              left: `${pos.left}px`,
-              top: `${pos.top}px`,
-              position: 'absolute',
-              right: 'auto',
-              margin: 0,
-              opacity: 1,
-            },
-            styles
-          ),
-        });
+    // If no pos available, we need to hide the element (offscreen to the left)
+    // This is done so we can measure the content before positioning it correctly.
+    const positioningStyle = pos
+      ? {
+          left: `${pos.left}px`,
+          top: `${pos.top}px`,
+          right: 'auto',
+        }
+      : {
+          right: `${window.innerWidth}px`,
```

Comment:
Thanks @psachs21 for explaining your finding in detail! Your fix makes sense basically, just one thing - Given you want to make the temporally menu location absolute and off-screen - Did you mean `left` here?

Reply:
It's currently positioned off the the left of the screen, but we can put it off to the right as well. Both are roughly equivalent.

Reply:
Yeah I thought the same - Like `right: `${window.innerWidth}px` or `left: 0`.

Reply:
^ I guess we'd choose either of those. Thanks!

Reply:
hmm, your right, I had a brain fart. Left would be the natural fit for me. I'll update the PR.